src/poliastro/twobody/orbit.py

```diff
@@ -610,6 +610,71 @@ def parabolic(
         ss = ClassicalState(attractor, p, 1.0 * u.one, inc, raan, argp, nu)
         return cls(ss, epoch, plane)
 
+    @classmethod
+    @u.quantity_input(alt=u.m, inc=u.rad, argp=u.rad, raan=u.rad, arglat=u.rad)
+    def frozen(
+        cls,
+        alt,
+        inc=Earth.critical_inclinations[0],
```

Comment:
If I understand correctly, your logic is:

1. If the provided argument of periapsis is close to the critical one, use the provided inclination and freeze the eccentricity
2. If not, but the provided inclination is close to the critical one, use it
3. Else, raise an error

Now I realize this was not as straightforward as imagined! (also because I had forgotten what frozen orbits actually were).

I think it would be nice to document this logic, and also add some mathematical background. According to [J. Hoogland master thesis "Earth frozen orbits: design, injection and stability"](https://repository.tudelft.nl/islandora/object/uuid:fc0a57ce-33df-4cd2-a61b-66e702cc9cf8/datastream/OBJ/download), and probably also Vallado (latest editions not easily available on the Internet), there are two cases of frozen orbits (page 20):

1. (And now I wish GitHub had LaTeX support) (1 - 5/4 * sin^2 i) == 0. There are two critical inclinations and they don't depend on the attractor.
2. Argument of periapsis = 90 deg (for every attractor) and eccentricity given by J2, J3, the radius of the attractor, the semimajor axis and the _chosen_ inclination.